Rakefile

```diff
@@ -28,22 +26,29 @@ task spec: :prepare_test_env
 
 desc "Prepare money-rails engine test environment"
 task :prepare_test_env do
-  Rake.application['app:db:drop:all'].invoke
-  Rake.application['app:db:create'].invoke if Rails::VERSION::MAJOR >= 5
-  Rake.application['app:db:migrate'].invoke
-  Rake.application['app:db:test:prepare'].invoke
+  load APP_RAKEFILE if File.exist?(APP_RAKEFILE)
+  invoke_db_task("drop")
+  invoke_db_task("create")
+  invoke_db_task("migrate")
+  invoke_db_task("test:prepare")
 end
 
 def run_with_gemfile(gemfile)
   Bundler.with_original_env do
-    begin
-      sh "BUNDLE_GEMFILE='#{gemfile}' bundle install --quiet"
-      Rake.application['app:db:create'].invoke
```

Comment:
before running the tests, the `prepare_test_env` is run so I don't think we need to do this here 🤔